### PR TITLE
Re-enable test case to ensure EmbeddingBag gradient computation works

### LINKED ISSUE #2215: embeddingbad error
reported by @taylanbil 
```
import torch
import torch.nn as nn
import torch_xla.core.xla_model as xm


device = xm.xla_device()
d = nn.EmbeddingBag(10, 10, mode="sum", sparse=False).to(device)
inp = torch.LongTensor([1, 5, 9]).to(device)
x = d(inp, offsets=torch.LongTensor([0]).to(device))
loss = x.sum()
loss.backward()
```

### PR DESCRIPTION
**Pull request by gmagogsfm**
This PR tests https://github.com/pytorch/pytorch/pull/40557 in pytorch/pytorch.

Closes #2215

### CODE REVIEW

File: test/test_operations.py [hunk 1]
```diff
@@ -1948,6 +1948,14 @@ def from_tensors(self, tensors):
     self.assertEqual(xdata.batch_sizes.device, torch.device('cpu'))
     self.assertEqual(xdata.data.device, xla_device)
 
+  def test_embedding_bag_backward(self):
```
Comment (dlibenzi) [inline test/test_operations.py:1951]: Hmm, the test does not do anything. Literally.
With LazyTensor nothing gets really executed there, modulo trips to CPU for eventually unsupported ops.
Do you have a test in PyTorch for this?

If not, maybe using the existing infrastructure, which checks against CPU:

https://github.com/pytorch/xla/blob/11ecce5a1f823d4c33f0f4e01232cff0a9e0147d/test/test_operations.py#L1023

**Comment by JackCaoG** [comment]
Thanks for making the change. we also disabled couple embeddingBag related test and I think your change might fix those tests. Could you give them a try? Searching for `EmbeddingBag` in https://github.com/pytorch/xla/blob/master/test/pytorch_test_base.py#L209 and delete those line will enable those tests.

File: test/test_operations.py (hunk 1 above)
Comment (gmagogsfm) [reply test/test_operations.py:1951]: We are indeed testing trips to CPU for an unsupported op (EmbeddingBag). There is no test in PyTorch for this as it is only a problem for pytorch/XLA. Let me know what you think.

Comment (dlibenzi) [reply test/test_operations.py:1951]: You should be able to write a test like that, so we compare the results.

File: test/pytorch_test_base.py [hunk 2]
```diff
@@ -208,7 +208,6 @@
     'TestNNDeviceTypeXLA': {
         'test_embedding_backward',  # sparse
         'test_embedding_dense_grad',  # slow
-        'test_EmbeddingBag_per_sample_weights_and_new_offsets',  # FIXME! UndefinedTensorImpl::_singleton
```
Comment (ailzhang) [inline test/pytorch_test_base.py:211]: Btw there're 3 more embeddingbag tests below, could you confirm they're still failing (are they different issues?) Thanks!

Comment (gmagogsfm) [reply test/pytorch_test_base.py:211]: Yes, they are still failing. Two of them are failing because they test sparse tensor behavior. The other fails due to a different runtime error.

**Comment by gmagogsfm** [comment]
> Thanks for making the change. we also disabled couple embeddingBag related test and I think your change might fix those tests. Could you give them a try? Searching for `EmbeddingBag` in https://github.com/pytorch/xla/blob/master/test/pytorch_test_base.py#L209 and delete those line will enable those tests.

Thanks, ended up enabling one of those tests instead of adding a new one.

File: test/test_operations.py (hunk 1 above)
Comment (gmagogsfm) [reply test/test_operations.py]: Resolving. Test case removed.

File: test/pytorch_test_base.py (hunk 2 above)
Comment (ailzhang) [reply test/pytorch_test_base.py:211]: Would you mind updating them with the latest error message? (The singleton issue should be fixed by your PR) Thanks

Comment (gmagogsfm) [reply test/pytorch_test_base.py:211]: Done. And had a nice surprise of finding an extra test that's passing but missed before

**Comment by gmagogsfm** [comment]
Pin removed.
